README.md

```diff
@@ -185,6 +187,10 @@ random:
    seed: -1 # A value of -1 will result in a seed derived from the system clock value
 
 input:
+  # Random data collection
+  random:
```

Comment:
As KPZ says probably best to rename this to be more descriptive, maybe something like "randomDataCollection" or "useRandomFrames" or "useRandomPositions" -- whichever you think fits best

Reply:
That's a fair comment. I've renamed it to randomDataCollection - good suggestion!